shotgun_api3/shotgun.py

```diff
@@ -409,7 +409,7 @@ def __init__(self,
         else:
             self.__ca_certs = os.environ.get('SHOTGUN_API_CACERTS')
 
-        self.base_url = (base_url or "").lower()
+        self.base_url = (base_url or "")
```

Comment:
There's some reasoning further down in the code, but I agree that we shouldn't risk the larger potential issues by changing it here.  I'm reverting this and making a more specific change to where it's particularly relevant and almost zero risk.